content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Quality Gates
+kind: documentation
+description: Learn how to use Quality Gates in your pipeline.
+is_beta: true
+further_reading:
+- link: "/continuous_integration/tests"
+  tag: "Documentation"
+  text: "Learn about Test Visibility"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="#" header="false" >}}
+Quality Gates are in beta.
+{{< /callout >}}
+
+Quality Gates allows you to gate your workflows based on signals in Datadog.
+For example, you can block code from being merged if it introduces new [flaky tests][8], even if the tests are passing.
```

Comment:
can we provide more examples to promote the feature better?

Reply:
... if it introduces new flaky tests or code style violations; issues that wouldn't normally fail your CI/CD pipelines and end up deployed to production.

Reply:
I have changed the sentence as Bryan mentioned. @liashenko Do you think that's ok on your side?